### ISSUE TITLE
Containers: Support ARM 32bit

### ISSUE THREAD

**Issue by JamesGreenaway** [microsoft/vscode-remote-release#1813]
VSCode: 1.40.0
Remote SSH: 1.47.2
Mac: 10.15.1
Remote OS: Raspbian Buster

I am attempting to run Remote - SSH inside a container running on a remote docker host. 
Steps to reproduce: 
1. add a basic `.devcontainer.json`: 

```
{
  "image": "node",
  "workspaceFolder": "/workspace",
  "workspaceMount": "src=remote-workspace,dst=/workspace,type=volume,volume-driver=local"
}
```
2. add the following to `settings.json`: 

```
"docker.host":"tcp://your-remote-machine-fqdn-or-ip-here:port"
```

3. Run 'Open folder in container'

Error message: 
```
...
...Exec format error
Command failed: docker exec -w /home/vscode/.vscode-server/bin/86405ea23e3937316009fc27c9361deee66ffbf5 -e SHELL=/bin/bash -e VSCODE_AGENT_FOLDER=/home/vscode/.vscode-server 9f04be659d8bc717879dc006e2f1b0e2209208bd513e6eb342f9f094910b063b /home/vscode/.vscode-server/bin/86405ea23e3937316009fc27c9361deee66ffbf5/server.sh --disable-user-env-probe --port 0
```

**Comment by chrmarti** [comment]
Related: #1716

**Comment by chrmarti** [comment]
Fixed with #1716.
